diff --git a/node_modules/react-native-progress-display/.DS_Store b/node_modules/react-native-progress-display/.DS_Store
new file mode 100644
index 0000000..7083c2c
Binary files /dev/null and b/node_modules/react-native-progress-display/.DS_Store differ
diff --git a/node_modules/react-native-progress-display/ios/RNProgressHud.podspec b/node_modules/react-native-progress-display/ios/RNProgressHud.podspec
index a644894..e52c830 100644
--- a/node_modules/react-native-progress-display/ios/RNProgressHud.podspec
+++ b/node_modules/react-native-progress-display/ios/RNProgressHud.podspec
@@ -6,13 +6,13 @@ Pod::Spec.new do |s|
   s.description  = <<-DESC
                   RNProgressHud
                    DESC
-  s.homepage     = ""
+  s.homepage     = "http://example.com"
   s.license      = "MIT"
   # s.license      = { :type => "MIT", :file => "FILE_LICENSE" }
   s.author             = { "author" => "author@domain.cn" }
   s.platform     = :ios, "7.0"
   s.source       = { :git => "https://github.com/author/RNProgressHud.git", :tag => "master" }
-  s.source_files  = "RNProgressHud/**/*.{h,m}"
+  s.source_files  = "*.{h,m}"
   s.requires_arc = true
 
 
