diff --git a/node_modules/react-native-gifted-chat/lib/GiftedAvatar.js b/node_modules/react-native-gifted-chat/lib/GiftedAvatar.js
index 266857c..45174d0 100644
--- a/node_modules/react-native-gifted-chat/lib/GiftedAvatar.js
+++ b/node_modules/react-native-gifted-chat/lib/GiftedAvatar.js
@@ -3,6 +3,7 @@ import React from 'react';
 import { Image, Text, TouchableOpacity, View, StyleSheet, } from 'react-native';
 import Color from './Color';
 import { StylePropType } from './utils';
+import FastImage from 'react-native-fast-image';
 const { carrot, emerald, peterRiver, wisteria, alizarin, turquoise, midnightBlue, } = Color;
 const styles = StyleSheet.create({
     avatarStyle: {
@@ -71,10 +72,10 @@ export default class GiftedAvatar extends React.Component {
                 return user.avatar([styles.avatarStyle, this.props.avatarStyle]);
             }
             else if (typeof user.avatar === 'string') {
-                return (<Image source={{ uri: user.avatar }} style={[styles.avatarStyle, this.props.avatarStyle]}/>);
+                return (<FastImage source={{ uri: user.avatar }} style={[styles.avatarStyle, this.props.avatarStyle]}/>);
             }
             else if (typeof user.avatar === 'number') {
-                return (<Image source={user.avatar} style={[styles.avatarStyle, this.props.avatarStyle]}/>);
+                return (<FastImage source={user.avatar} style={[styles.avatarStyle, this.props.avatarStyle]}/>);
             }
         }
         return null;
diff --git a/node_modules/react-native-gifted-chat/lib/MessageImage.js b/node_modules/react-native-gifted-chat/lib/MessageImage.js
index dd0000e..cb08988 100644
--- a/node_modules/react-native-gifted-chat/lib/MessageImage.js
+++ b/node_modules/react-native-gifted-chat/lib/MessageImage.js
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { Image, StyleSheet, View, } from 'react-native';
+import FastImage from 'react-native-fast-image';
 // TODO: support web
 // @ts-ignore
 import Lightbox from 'react-native-lightbox';
@@ -27,7 +28,7 @@ export default class MessageImage extends Component {
           <Lightbox activeProps={{
                 style: styles.imageActive,
             }} {...lightboxProps}>
-            <Image {...imageProps} style={[styles.image, imageStyle]} source={{ uri: currentMessage.image }}/>
+            <FastImage {...imageProps} style={[styles.image, imageStyle]} source={{ uri: currentMessage.image }}/>
           </Lightbox>
         </View>);
         }
